backend/workflow_manager/endpoint/destination.py

```diff
@@ -155,20 +155,19 @@ def handle_output(
         file_name: str,
         file_hash: FileHash,
         workflow: Workflow,
+        input_file_path: str,
```

Comment:
Why this changed from Optional to str?

Reply:
To fix a code smell reported by mypy extension. Always an str is passed here